bindings/go/scip/symbol.go

```diff
@@ -22,10 +21,13 @@ func IsLocalSymbol(symbol string) bool {
 	return strings.HasPrefix(symbol, "local ")
 }
 
+func isSimpleIdentifierCharacter(c rune) bool {
+	return ('0' <= c && c <= '9') || ('a' <= c && c <= 'z') || ('A' <= c && c <= 'Z') || c == '$' || c == '+' || c == '-' || c == '_'
```

Comment:
```suggestion
	return c == '_' || c == '+' || c == '-' || c == '$' || ('a' <= c && c <= 'z') || ('A' <= c && c <= 'Z') || ('0' <= c && c <= '9')
```

nit: Re-arranging to match up with order in doc comment in spec